### PR TITLE
Super minimal no-JS landing page

### LINKED ISSUE #1: Add basic page showing all projects
https://ben.balter.com/2015/06/11/using-github-pages-to-showcase-your-orgs-open-source-efforts/

### PR DESCRIPTION
**Pull request by PayasR**
Fixes #1

### CODE REVIEW

**Comment by PayasR** [comment]
image

File: docs/index.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" ./kkkkk>
```
Comment (frewsxcv) [inline docs/index.html:4]:
```suggestion
  <meta charset="UTF-8">
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "type": "git",
+    "url": "git+https://github.com/PayasR/georust.github.io.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/PayasR/georust.github.io/issues"
+  },
+  "homepage": "https://github.com/PayasR/georust.github.io#readme",
```
Comment (frewsxcv) [inline package.json:19]:
```suggestion
    "url": "git+https://github.com/georust/georust.github.io.git"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "bugs": {
    "url": "https://github.com/georust/georust.github.io/issues"
  },
  "homepage": "https://github.com/georust/georust.github.io#readme",
```

**Review by frewsxcv — APPROVED** [review]
looks great! thank you so much!

**Pull request merged**